src/profiler.h

```diff
@@ -58,54 +58,71 @@ class Profiler
 	void getPage(GraphValues &o, u32 page, u32 pagecount);
 
 
+	void graphSet(const std::string &id, float value)
+	{
+		MutexAutoLock lock(m_mutex);
+		m_graphvalues[id] = value;
+	}
 	void graphAdd(const std::string &id, float value)
 	{
 		MutexAutoLock lock(m_mutex);
-		std::map<std::string, float>::iterator i =
-				m_graphvalues.find(id);
-		if(i == m_graphvalues.end())
-			m_graphvalues[id] = value;
+		auto it = m_graphvalues.find(id);
+		if (it == m_graphvalues.end())
+			m_graphvalues.emplace(id, value);
 		else
-			i->second += value;
+			it->second += value;
 	}
 	void graphGet(GraphValues &result)
```

Comment:
Observation: This function is more accurately `graphPop`. `graphGet` does not indicate that the value is eaten. The function could either be renamed or a comment added to explain this behaviour.

I know the functionality is not touched by this PR, hence feel free to reject this suggestion if it is not suited for this PR.

Reply:
Sure, I can rename. There are only two usages.